src/components/image-uploader/image-uploader.tsx

```diff
@@ -37,7 +37,7 @@ export type ImageUploaderProps = {
   deletable?: boolean
   capture?: InputHTMLAttributes<unknown>['capture']
   onPreview?: (index: number, item: ImageUploadItem) => void
-  beforeUpload?: (file: File[]) => Promise<File[]> | File[]
+  beforeUpload?: (file: File, files: File[]) => Promise<File> | File
```

Comment:
返回值改成是 `File | null` 吧，明确把 `null` 标出来，有些项目的 ts 会开 strictNullChecks 的

```suggestion
  beforeUpload?: (file: File, files: File[]) => Promise<File | null> | File | null
```